lib/models/msal_user.dart

```diff
@@ -15,12 +15,16 @@ class MsalUser {
   /// Token expiration time in "MillisecondsSinceEpoch".
   final int tokenExpiresOn;
 
+  /// The raw id token if it's returned by the service or nil if no id token is returned.
```

Comment:
Your comment has the wrong information. how `final String idToken` can be `nil`? it will have a value or blank.

Reply:
already update it.